### PR TITLE
Fix undocumented methods in the List class ( #878)

### PR DESCRIPTION
**Pull request by PureFox48**
Also noted that the ```add``` method returns the added item.

### CODE REVIEW

File: doc/site/modules/core/list.markdown [hunk 1]
```diff
@@ -173,3 +185,13 @@ var other = ["d", "e", "f"]
 var combined = letters + other
 System.print(combined)  //> [a, b, c, d, e, f]
 </pre>
+
+### **\***(count) operator
```
Comment (ChayimFriedman2) [inline doc/site/modules/core/list.markdown:189]: Should be two asterisks, not three.

**Review by ChayimFriedman2 — CHANGES_REQUESTED** [review]

File: doc/site/modules/core/list.markdown (hunk 1 above)
Comment (PureFox48) [reply doc/site/modules/core/list.markdown:189]: I thought that if you wanted a literal asterisk you had to escape it with a back-slash in Github markdown?

Comment (ChayimFriedman2) [reply doc/site/modules/core/list.markdown:189]: Oh my bad, I thought that this is operator `\` (now I recall that the operator is reverse, `/`) 😄.

**Review by ChayimFriedman2 — APPROVED** [review]

File: doc/site/modules/core/list.markdown (hunk 1 above)
Comment (PureFox48) [reply doc/site/modules/core/list.markdown:189]: No problem. I had 5 asterisks to begin with before I realized the middle one needed escaping!

**Pull request merged**